packages/opentelemetry-exporter-collector/src/platform/node/types.ts

```diff
@@ -38,3 +47,14 @@ export interface TraceServiceClient extends grpc.Client {
     callback: Function
   ) => {};
 }
+
+/**
+ * Metric Service Client for sending metrics
+ */
+export interface MetricsServiceClient extends grpc.Client {
+  export: (
+    request: any,
+    metadata: grpc.Metadata | undefined,
+    callback: Function
+  ) => {};
+}
```

Comment:
On second thought, should I just make this "ServiceClient" because this is the same as TraceServiceClient

Reply:
Yes I think thats a better choice rather than having two identical interfaces